requirements.in

```diff
@@ -1,28 +1,31 @@
-flask
-flask-appconfig>=0.10
-flask-security
-flask-sqlalchemy
-flask-restful
-flask-cors
-jsl
+tenacity
+Werkzeug
+Flask_Cors
+Flask_SQLAlchemy
+Flask
+flask_appconfig
+py_lru_cache
+lxml
+requests
+SQLAlchemy
+raven
+pydantic
+pytest
```

Comment:
Do not have test dependencies in this file. Create a separate test requirement.in that takes all present in main requirement.in and add test deps on top. Refer to commit https://github.com/fabric8-analytics/fabric8-analytics-utils/commit/21ea911551248a4d485d1ac3e42c1ad7fa6849a0